dbquery/custom-query/tests/Test.js

```diff
@@ -0,0 +1,74 @@
+const fs = require("fs");
+const path = require("node:path");
+const {
+  deployAndRun,
+  stepzen,
+  getTestDescription,
+} = require("../../../tests/gqltest.js");
+
+testDescription = getTestDescription("snippets", __dirname);
+
+// Read the GraphQL operations from the operations file
+const requestsFile = path.join(path.dirname(__dirname), "operations.graphql");
+const requests = fs.readFileSync(requestsFile, "utf8").toString();
+
+describe(testDescription, function () {
+
+  const tests = [
+    {
+      label: "fetch all customers",
+      query: requests,
+      operationName: "GetAllCustomers",
+      variables: {},
+      expected: {
+        getAllCustomers: [
+          { id: "1", name: "Lucas Bill                                        ", email: "lucas.bill@example.com                            " },
```

Comment:
Why empty space is coming for name and email

Reply:
Its how in the DB data resides